src/todo/UI.lua

```diff
@@ -62,51 +68,59 @@ function todo.create_task_table(frame, player)
 
     local table = scroll.add({
         type = "table",
+        style = "todo_table_default",
         name = "todo_task_table",
         column_count = 6
     })
 
     table.add({
         type = "label",
+        style = "todo_label_default",
         name = "todo_title_done",
         caption = {"", {"todo.title_done"}, "   "}
     })
 
     table.add({
         type = "label",
+        style = "todo_label_default",
         name = "todo_title_task",
         caption = {"todo.title_task"}
     })
 
     table.add({
         type = "label",
+        style = "todo_label_default",
         name = "todo_title_assignee",
         caption = {"todo.title_assignee"}
     })
 
     table.add({
         type = "label",
+        style = "todo_label_default",
         name = "todo_title_up",
         caption = ""
     })
 
     table.add({
         type = "label",
+        style = "todo_label_default",
         name = "todo_title_down",
         caption = ""
     })
 
     table.add({
         type = "label",
+        style = "todo_label_default",
         name = "todo_title_edit",
         caption = ""
     })
 
     return table
 end
 
-function todo.create_add_edit_frame(player)
+function todo.create_add_edit_frame(player, task)
     local gui = player.gui.center
+    task = task or nil
```

Comment:
Not sure if this is really necessary, but better safe than sorry, I guess?

Reply:
I did an experiment once and it should not be necessary, but I cannot recall the details anymore. So I aggree.. better safe than sorry :D